deploy/modules/statistics/main.tf

```diff
@@ -1,20 +1,30 @@
-#################################
+module "globals" {
+  source = "../../modules/core/globals"
```

Comment:
You should use ${path.module}

Reply:
Does this means that we create an additional ssh key (It is part of the globals module..)